flexmeasures/cli/tests/conftest.py

```diff
@@ -209,3 +210,46 @@ def storage_schedule_sensors(
     db.session.commit()
 
     yield power_capacity.id, storage_efficiency.id
+
+
+@pytest.fixture(scope="module")
+def add_asset_with_children(db, setup_roles_users):
+    test_supplier_user = setup_roles_users["Test Supplier User"]
+    parent_type = GenericAssetType(
+        name="parent",
+    )
+    child_type = GenericAssetType(name="child")
+
+    db.session.add_all([parent_type, child_type])
+
+    parent = GenericAsset(
+        name="parent",
+        generic_asset_type=parent_type,
+        account_id=test_supplier_user,
```

Comment:
This seems to put a `User` Id where an `Account` Id should be. Or am I mistaken?

Maybe this is not an error because the Id also exists as an `Account`.

Reply:
You're right! As you mentioned, this doesn't break the test at it already exists an account with Id = test_supplier_user. I've updated the code to use the account id instead.